fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -167,6 +167,12 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         browserToolbarView.view.addPageAction(readerModeAction)
 
         initTranslationsAction(context, view)
+        if (!IncompleteRedesignToolbarFeature(context.settings()).isEnabled) {
+            browserToolbarView.view.hidePageActionSeparator()
+        } else {
+            browserToolbarView.view.showPageActionSeparator()
+        }
+
```

Comment:
It's kinda a shame that we are adding this in between all the init action functions. I am wondering if this really needed to happen in this particular order or could we just call hide/show before or after these chains of init()

Reply:
do you have any particular concern, @gabrielluong? The navbar init happens even [earlier](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt#446) than this call, I wonder if it might be a problem.

If it's not, I would just moving this code into `BaseBrowserFragment`, since there is already feature based ui adjustments. Would be nice to have it in one place.

Reply:
No you are right @gabrielluong . Shouldn't have been in between those initializations. I moved it to `BaseBrowserFragment`